contracts/ToucanRegenBridge.sol

```diff
@@ -19,6 +19,7 @@ contract ToucanRegenBridge is Pausable, AccessControl {
     // ----------------------------------------
 
     bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
+    bytes32 public constant TOKEN_ISSUER_ROLE = keccak256("TOKEN_ISSUER_ROLE");
```

Comment:
This commit's changes look good, but again the commit message could use improvement.

Most importantly (as always) it should contain an explanation of *why* this change is being done.  It could mention that we are doing this to unify access control into a single mechanism, and include the reasons our auditors gave us for why this is beneficial.

It could also explain why this change allows us remove an event and an existing function.

Finally, either this commit should come before the variable rename commit (which is currently the one before it), otherwise the rename is premature, or actually there is an argument that the two commits could be squashed together because they are strongly linked.

Reply:
I'd rather have them separate, tbh. It's easier to distinguish what happened this way. 

I also think it makes sense for the `bridgeAdmin -> tokenIssuer` rename to occur before this because that was a refactor that made it easier for me to know who's who when modifying the tests in the commit that adds `TOKEN_ISSUER_ROLE`.

That said, I can see how the messages could describe all of this better.